app/src/main/java/com/kks/nimblesurveyjetpackcompose/model/response/IncludedResponse.kt

```diff
@@ -48,11 +49,20 @@ fun IncludedQuestionResponse.toSurveyQuestion() =
         title = attributes?.text.orEmpty().trim(),
         displayOrder = attributes?.displayOrder ?: 0,
         shortText = attributes?.shortText.orEmpty(),
-        pick = attributes?.pick.orEmpty(),
+        pick = attributes?.pick.toSurveyQuestionPickType(),
         questionDisplayType = attributes?.displayType.orEmpty().getQuestionDisplayType(),
         answers = emptyList()
     )
 
+private const val SINGLE_PICK = "one"
```

Comment:
Shall we move this to the top of the file? It would be good to have common organization of our code 😉

Reply:
Also, for resources with snake case, we should follow the same naming pattern, e.g., `PICK_SINGLE`. What do you think?